webdriver/tests/bidi/browsing_context/dom_content_loaded/dom_content_loaded.py

```diff
@@ -142,22 +143,33 @@ async def on_event(method, data):
 
 
 @pytest.mark.parametrize("type_hint", ["tab", "window"])
-async def test_new_context(bidi_session, subscribe_events, wait_for_event, wait_for_future_safe, type_hint):
+async def test_new_context_not_emitted(bidi_session, subscribe_events,
+      wait_for_event, wait_for_future_safe, type_hint):
     await subscribe_events(events=[DOM_CONTENT_LOADED_EVENT])
 
-    on_entry = wait_for_event(DOM_CONTENT_LOADED_EVENT)
-    new_context = await bidi_session.browsing_context.create(type_hint=type_hint)
-    event = await wait_for_future_safe(on_entry)
+    # Track all received browsingContext.domContentLoaded events in the events array
+    events = []
 
-    assert_navigation_info(
-        event, {"context": new_context["context"], "url": "about:blank"}
+    async def on_event(method, data):
+        events.append(data)
+
+    remove_listener = bidi_session.add_event_listener(
+        DOM_CONTENT_LOADED_EVENT, on_event
     )
-    assert event["navigation"] is not None
+
+    await bidi_session.browsing_context.create(type_hint=type_hint)
+
+    # In the future we can wait for "browsingContext.domContentLoaded" event instead.
```

Comment:
Could clarify this comment? I didn't get why we would need to wait for domContentLoaded event here?

Reply:
copy-paste
```suggestion
```